loki/README.md

```diff
@@ -0,0 +1,124 @@
+# Installing a custom loki-stack in Kyma
+
+## Overview
+
+The Kyma Loki component brings limited configuration options in contrast to the upstream [loki-stack]('https://github.com/grafana/helm-charts/tree/main/charts/loki-stack') chart. Modifications might be reset at next upgrade cycle.
+
+An alternative can be a parallel installation of the upstream chart offering all customization options. The following instructions outline how to achieve such installation in co-existence to the Kyma stack.
+
+**CAUTION:** This example will use the Grafana Loki version which is distributed under AGPL-3.0 only and might not be free of charge for commercial usage.
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- Kubectl > 1.22.x
+- Helm 3.x
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_LOKI_EXAMPLE_NS="{namespace}"
+    ```
+
+2. Export the Helm release name that you want to use. It can be any name, but be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
+    ```bash
+    export HELM_RELEASE_NAME="{release-name}"
+    ```
+
+3. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    helm repo update
+    ```
+
+### Install the loki-stack
+
+Run the Helm upgrade command, which installs the chart if not present yet.
+ ```bash
+helm upgrade --install -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/values.yaml
```

Comment:
Let's add `--create-namespace` since it is not added previously or the existence of the namespace described as a prerequisite.

Reply:
Then we should add it for kiali tutorial as well. Is it important if istio is enabled or not? I guess it is important for the push URL (http|s) so maybe we should have a prerequisite defined instead?

Reply:
For Kiali we use the `--create-namespace` and enable the sidecar using the Pod label. It would work with or without Istio (here and for Kiali). The https redirect should be transparent with the istio-proxy and no changes required.

Reply:
`--create-namespace` parameter added for all Helm installation commands